packages/@aws-cdk/aws-elasticloadbalancingv2/lib/nlb/network-listener.ts

```diff
@@ -112,25 +132,51 @@ export class NetworkListener extends BaseListener implements INetworkListener {
 
     this.loadBalancer = props.loadBalancer;
 
-    (props.defaultTargetGroups || []).forEach(this._addDefaultTargetGroup.bind(this));
+    if (props.defaultAction && props.defaultTargetGroups) {
+      throw new Error('Specify at most one of \'defaultAction\' and \'defaultTargetGroups\'');
+    }
```

Comment:
can't find tests. sorry, if they're somewhere and I just can't spot them. I blame the PR size 😉